Podcast/SearchTableViewController.swift

```diff
@@ -23,6 +24,8 @@ enum SearchType {
             return "Series"
         case .people:
             return "People"
+        case .itunes:
```

Comment:
When trying to subscribe to a series 
<img width="978" alt="screen shot 2017-11-01 at 11 38 55 am" src="https://user-images.githubusercontent.com/14966713/32282963-70a1b4ba-bef9-11e7-8cec-3fe7f3dd2d86.png">